reference/pipeline/expressions/index.md

```diff
@@ -0,0 +1,214 @@
+---
+layout: single
+title:  "Pipeline Expression Reference"
+sidebar:
+  nav: reference
+---
+
+{% include toc %}
+
+Pipeline expressions allow you to reference arbitrary values about the
+state of your system in the execution of your pipelines. For more information
+about how pipeline expressions work, see the
+[overview](/guides/user/pipeline/expressions/).
+
+## Syntax elements
+
+Each pipeline expression is made of `$` followed by opening/closing brackets:
```

Comment:
I think it is worth noting here that expressions cannot be nested. A common mistake we see is people trying to use  ${ something ${ something else } } and not having their expressions evaluated.